README.md

```diff
@@ -100,6 +100,8 @@ echo 'terraform { backend "s3" {} }' > conf.tf
 terraform init -reconfigure \
     -backend-config="bucket=terraform-state-bucket" \
     -backend-config="key=states/terraform.tfstate" \
+    -backend-config="encrypt=1" \
+    -backend-config="kms_key_id=YOUR_KMS_KEY_ID" \
```

Comment:
Is this always required? It uses the default key if not specified, right?

Reply:
i have simplified the examples to reflect the improved defaults.